docs/guides-publish.md

```diff
@@ -18,50 +18,66 @@ This guide will show you how to publish your app to an [aragonPM](/docs/package-
 We'll start from the [React boilerplate](https://github.com/aragon/aragon-react-boilerplate).
 
 ```sh
-npx create-aragon-app app react
+npx create-aragon-app app
 ```
 
 This will create a new directory named `app`, with everything you need.
 
+Then to interact with aragonPM we will use the [`aragon apm`](/docs/cli-apm-commands) commands.
+
 ## Introduction to environments
 
-This app has 3 environments defined in `arapp.json`:
+This app has 3 environments defined:
 
-| Environment   | Network   |
-|---            |---        |
-| default       | localhost |
-| staging       | rinkeby   |
-| production    | mainnet   |
+| Environment | Network   |
+| ----------- | --------- |
+| default     | localhost |
+| staging     | rinkeby   |
+| production  | mainnet   |
 
 Is a prerequisite to have a ENS Registry address defined.
 
-Environments are defined in `arapp.json`, for example `staging` points to:
-- an ENS registry: `0x314159265dd8dbb310642f98f50c066173c1259b`
-- an APM registry: `open.aragonpm.eth`
-- an APM repository: `app`
-- an Ethereum network: `rinkeby`
-- an Ethereum websockets provider: `wss://rinkeby.eth.aragon.network/ws` - to **read** from the blockchain
+Environments are defined in [`arapp.json`](/docs/cli-global-confg#the-arappjson-file), for example `staging` points to:
+
+- An ENS registry: `0x98df287b6c145399aaa709692c8d308357bc085d`
+- An app name (repository and registry of aragonPM): `app.open.aragonpm.eth`
+- An Ethereum websockets provider: `wss://rinkeby.eth.aragon.network/ws` - to **read** from the blockchain
+- An Ethereum network: `rinkeby`
 
 The `rinkeby` network is further defined in `truffle.js`, an has an Ethereum provider (to **write** to the blockchain), which define:
-- an address: `https://rinkeby.infura.io`
-- an Ethereum account: `0xb41...6eE7` (which is the first account generated from the `DEFAULT_MNEMONIC` variable, to use a different account see the [troubleshooting guide](/docs/guides-faq.html#set-a-private-key)
 
-## Major version: content + contract
+- An address: `https://rinkeby.infura.io`
+- An Ethereum account: `0xb41...6eE7` (which is the first account generated from the `DEFAULT_MNEMONIC` variable, to learn how to use a different account follow the [troubleshooting guide](/docs/guides-faq.html#set-a-private-key))
+
+> **Note**<br>
+> The `default` environment which points to `localhost` does not have an ENS Registry address specified because aragonCLI will default the value to `0xB9462EF3441346dBc6E49236Edbb0dF207db09B7` (the ENS Registry pre-deployed on the local development chain).
+
+## Publish a major version: content + contract
+
+To publish on aragonPM we will use [`aragon apm publish`](/docs/cli-apm-commands#aragon-apm-publish) command.
 
 Command:
 
 ```sh
-npm run publish:major --environment staging
+npx aragon apm publish major --environment staging
```

Comment:
I think we should avoid using staging, as AFAIK we don't have `open.aragonpm.eth` set up on it yet. We should use `rinkeby` instead, with an app name that points to `<app name>.open.aragonpm.eth`, since that would most closely mirror someone's app right now.

Reply:
That's actually rinkeby, when I updated the `arapp.json` file long ago I wasn't aware of the a1 staging environmnet.

I opened https://github.com/aragon/aragon-react-boilerplate/pull/56